File: server/router/admin/gost_client_proxy.go
package admin

import (
	"github.com/gin-gonic/gin"
	"server/controller/admin/gost_client_proxy"
	"server/global"
	"server/router/middleware"
)

func InitGostClientProxy(group *gin.RouterGroup) {
	g := group.Group("gost/client/proxy", middleware.Auth(global.Jwt), middleware.AuthAdmin())
	g.POST("config", gost_client_proxy.Config)
	g.POST("delete", gost_client_proxy.Delete)
	g.POST("page", gost_client_proxy.Page)
}

File: web/src/api/admin/gost_client_proxy.js
import request from "../../request/index.js";

const baseUrl = '/v1/admin/gost/client/proxy'

export const apiAdminGostClientProxyList = () => {
    return request.request({
        url: `${baseUrl}/list`,
        method: 'POST',
    })
}

export const apiAdminGostClientProxyPage = (data) => {
    return request.request({
        url: `${baseUrl}/page`,
        method: 'POST',
        data
    })
}

export const apiAdminGostClientProxyConfig = (data) => {
    return request.request({
        url: `${baseUrl}/config`,
        method: 'POST',
        data
    })
}

export const apiAdminGostClientProxyDelete = (data) => {
    return request.request({
        url: `${baseUrl}/delete`,
        method: 'POST',
        data
    })
}